AppVeyor System.Version error

Just tried to do an AppVeyor build, and got the following error:

```
Downloading appveyor.ps1 (5,609 bytes)...100%
.\appveyor.ps1 "bootstrap" -verbose
VERBOSE: copy the package to sublime text Packages directory
Cannot convert value "st2-0.10.6" to type "System.Version". Error: "Input string was not in a correct format."
```

Looks like there is an odd tag on the https://github.com/randy3k/UnitTesting repo.  Am I doing something wrong?

@randy3k I'm having this issue too. It seems to be due to a tag `st2-0.10.6`. Maybe removing it would sove the issue.

Details:

```
2018-07-04T04:51:54.9840436Z ##[error]% : Cannot convert value "st2-0.10.6" to type "System.Version". Error: "Input string was not in a correct format."
At D:\a\1\s\appveyor.ps1:40 char:61
+ ...           | where-object {$_ -notmatch "\^"} |%{[System.Version]$_} `
+                                                   ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [ForEach-Object], RuntimeException
    + FullyQualifiedErrorId : InvalidCastParseTargetInvocation,Microsoft.PowerShell.Commands.ForEachObjectCommand
 

2018-07-04T04:51:56.3456108Z ##[error]Process completed with exit code 1 and had 1 error(s) written to the error stream.
```

--- a/sbin/appveyor.ps1
+++ b/sbin/appveyor.ps1
@@ -37,7 +37,8 @@ function Bootstrap {
             } elseif (${env:SUBLIME_TEXT_VERSION} -eq 3) {
                 # the latest tag
                 $UNITTESTING_TAG = git ls-remote --tags $UT_URL | %{$_ -replace ".*/(.*)$", '$1'} `
-                        | where-object {$_ -notmatch "\^"} |%{[System.Version]$_} `
+                        | where-object {$_ -match "^[0-9]+\.[0-9]+\.[0-9]+$"} `
+                        | %{[System.Version]$_} `
                         | sort | select-object -last 1 | %{ "$_" }
             }
         } else {

